mlflow/johnsnowlabs/__init__.py

```diff
@@ -448,9 +448,9 @@ def _save_jars_and_lic(dst_dir, store_license=False):
 
     suite = get_install_suite_from_jsl_home(False)
     if suite.hc.get_java_path():
-        shutil.copyfile(suite.hc.get_java_path(), deps_data_path / "hc_jar.jar")
+        shutil.copy2(suite.hc.get_java_path(), deps_data_path / "hc_jar.jar")
```

Comment:
/ai are there any risks if we replace `shutil.copyfile` with `shutil.copy2`?

Reply:
For what it's worth, the `log_artifacts` (plural) method uses [shutil.copytree](https://docs.python.org/3/library/shutil.html#shutil.copytree), and _that_ uses `copy2` under the hood. So if there were side effects, it would probably already be apparent.